microsoftgraph/docs-conceptual/overview.md

```diff
@@ -15,16 +13,23 @@ ms.reviewer: maisarissi
 The Microsoft Graph PowerShell SDK acts as an API wrapper for the Microsoft Graph APIs, exposing the entire API set for use in PowerShell. It contains a set of cmdlets that helps you manage identities at scale from automating tasks to managing users in bulk using Azure Active Directory (Azure AD).
 It will help administer every Azure AD feature that has an API in Microsoft Graph.
 
-The Microsoft Graph PowerShell SDK is the replacement for the Azure AD PowerShell module and is recommended for interacting with Azure AD.
+The commands in Microsoft Graph PowerShell are autogenerated from the Microsoft Graph API schema making it easier to get faster updates and functionality. The cmdlet reference content is also autogenerated from the API reference.
```

Comment:
This is fine. I like the transparency and additional info/context for the customers.